Resources/Prototypes/Actions/emotes.yml

```diff
@@ -0,0 +1,35 @@
+- type: emote
+  id: Flip
+  name: chat-emote-name-flip
+  chatMessages: ["chat-emote-msg-flip"]
+  chatTriggers:
+  - flip
```

Comment:
These need to be updated, we have message suffix trimming on EE, so you don't have to duplicate every chat trigger (iirc that is)